neqo-transport/src/packet/mod.rs

```diff
@@ -257,7 +258,8 @@ impl PacketBuilder {
     /// Returns true if padding was added.
     pub fn pad(&mut self) -> bool {
         if self.padding && !self.is_long() {
-            self.encoder.pad_to(self.limit, 0);
+            self.encoder
+                .pad_to(self.limit, FRAME_TYPE_PADDING.try_into().unwrap());
```

Comment:
I can back this out, but I found it weird that we hard-coded the frame type here.

Reply:
Can you use `Frame::Padding(1).get_type()`?  That should optimize well enough.

Reply:
Also returns `u64`, so would still need to convert.

Reply:
Ah, of course.  Which is of course why I just put a raw 0 there before.  What you have is totally fine.